v0/sources/shopify/util.js

```diff
@@ -66,13 +67,26 @@ const createPropertiesForEcomEvent = message => {
     "root",
     PRODUCT_MAPPING_EXCLUSION_FIELDS
   );
-
   mappedPayload.products = productsList;
   return mappedPayload;
 };
 
+const extractEmailFromPayload = event => {
+  const flattenedPayload = flattenJson(event);
+  let email;
+  Object.entries(flattenedPayload).some(([key, value]) => {
+    if (key.includes("email") || key.includes("Email")) {
```

Comment:
Is this the correct way to identify the email as fields like `emailSubject`, `badEmail` etc will also get matched?

Reply:
Fixed